Cannot resolve package

Our custom bundler script couldn't handle this package. After some investigation I discovered a typo in the `package.json` which I've fixed in #529 

Current Behavior:
```
PS C:\Users\patr\Code\yarn-test> node
Welcome to Node.js v16.13.1.
Type ".help" for more information.
> require.resolve('./node_modules/@duckdb/duckdb-wasm');
Uncaught Error: Cannot find module './node_modules/@duckdb/duckdb-wasm'
Require stack:
- <repl>
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.resolve (node:internal/modules/cjs/helpers:108:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '<repl>' ]
```

Expected Behavior:
```
> require.resolve('./node_modules/@duckdb/duckdb-wasm');
'C:\\Users\\patr\\Code\\yarn-test\\node_modules\\@duckdb\\duckdb-wasm\\dist\\duckdb-browser.cjs'
```

--- a/packages/duckdb-wasm/package.json
+++ b/packages/duckdb-wasm/package.json
@@ -81,8 +81,8 @@
         "dist",
         "!dist/types/test"
     ],
-    "main:": "dist/duckdb-browser.cjs",
-    "module:": "dist/duckdb-browser.mjs",
+    "main": "dist/duckdb-browser.cjs",
+    "module": "dist/duckdb-browser.mjs",
     "types": "dist/duckdb-browser.d.ts",
     "jsdelivr": "dist/duckdb-browser.cjs",
     "unpkg": "dist/duckdb-browser.mjs",

